src/main/java/com/moodybluez/enterprise/MoodyBluezController.java

```diff
@@ -14,4 +14,9 @@
     public String index() {
         return "index";
     }
+    @RequestMapping("/metrics")
+    public String metrics() {
+        return "metrics";
+    }
+
```

Comment:
I added an endpoint to house the metrics endpoint.

Reply:
Is there a matching HTML page?   If not, this is incomplete functionality.